templates/core/about/builds.html

```diff
@@ -74,29 +81,25 @@ <h4>Write attempt on read-only directories</h4>
         passed in the environment variable <code>OUT_DIR</code>.
     </p>
 
-    <h4>Hitting sandbox limits</h4>
+    <h4>Hitting resource limits</h4>
     <p>
-        All the builds on Docs.rs are executed inside a sandbox with limited
-        resources. If a build fails because it hit limits please
-        <a href="{{ docsrs_repo | safe }}/issues/new/choose">open an issue</a>
-        to get them increased for your crate.
-        Since our build agent has finite resources, we have to consider each case individually.
-        However, network access will not be enabled for any crate.
-        The current limits are:
+        All the builds are executed inside a sandbox with limited resources. The current limits are:
     </p>
 
     {{ macros::crate_limits(limits=limits) }}
 
-    <h4>Something else went wrong</h4>
     <p>
-        Please <a href="{{ docsrs_repo | safe }}/issues">file an issue</a> if you have trouble building a crate on Docs.rs.
+        If your build fails because it these limits, please
+        <a href="{{ docsrs_repo | safe }}/issues/new/choose">open an issue</a>
+        to get them increased for your crate.
+        Since our build agent has finite resources, we have to consider each case individually.
+        However, network access will not be enabled for any crate.
     </p>
 
-    <h3>Requesting rebuilds</h3>
+    <h4>Other failures and requesting rebuilds</h4>
     <p>
-        Some build failures can be fixed by Docs.rs.
-        For example, your crate may fail to build because of a missing dependency or a bug in Docs.rs.
-        You can request a rebuild by <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>.
+        If you encounter build failures that do not fall under these criteria, please <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>.
+        Some build failures can be fixed by Docs.rs, e.g., by issuing a rebuild after a bug has been fixed.
```

Comment:
This change no longer says 'a bug in Docs.rs' and just says 'a bug'. This is a change in meaning - rebuilds can't fix a bug in the crate itself.